src/core/retargeting_engine/python/deviceio_source_nodes/message_channel_config.py

```
@@ -0,0 +1,70 @@
+
+    name: str
+    channel_uuid: bytes
+    channel_name: str = ""
+    max_message_size: int = 64 * 1024
+    outbound_queue_capacity: int = 256
+
+    def create_nodes(self) -> tuple[MessageChannelSource, MessageChannelSink]:
+        if len(self.channel_uuid) != 16:
+            raise ValueError(
+                "MessageChannelConfig.channel_uuid must be exactly 16 bytes"
+            )
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)